samples/auth-user-sample/.eslintrc.yml

```diff
@@ -0,0 +1,3 @@
+---
+env:
```

Comment:
This should be moved to the `test/` directory.  We don't want ambient mocha allowed in the directory with the source itself

Reply:
Done